### ISSUE TITLE
Disable thumbnail creation and/or respect .nomedia file

### ISSUE THREAD

**Issue by bhe69** [zhanghai/MaterialFiles#366]
First of all I'd like to tell you that I absolute love Material Files.

I have a proposal for a new option or function in Material Files:

Please add an option to disable the creation of thumbnails for files. Sometimes there a reasons why the creation of thumbnails is unwanted. The user should have the possibility to disable the creation of thumbnails in the settings menu, (possibly with an option to delete already created thumbnails).

This proposal could be combined the detection of a ".nomedia" file. If a ".nomedia" file exists in a folder, no thumbnails should be created for files in this folder.

**Comment by zhanghai** [comment]
A `.nomedia` means the files under that directory should be ignored when gallery is indexing media files, however I don't think it mandates no thumbnail/preview when the user themself is actively viewing images under that directory.

Why do you dislike thumbnails? If it's out of privacy concern, Material Files actually stores thumbnail files inside its app data on device internal storage, meaning no other app can read it unless the system is compromised/rooted.

Meanwhile for the same reason, you can just clear cache data for this app in Settings' app detail page if you want to delete thumbnails.

**Comment by bhe69** [comment]
Yes, the main reason is privacy concern, further are user's choice and taking up storage space. It's good to know that the thumbnails are stored (somewhat) privately, however with root access you can undelete files even though you've cleared the cache. Hence an option to disable thumbnail creation at all would be great!

**Comment by GithbUser** [comment]
@bhe69 
We can disable thumbnail creation by adding a .nopreview file just like .nomedia file in mixplorer  and i think you will also want this enhancement in material files
 @zhanghai we want that material files should use this for stop thumbnail creation

.nosearch makes the folder content not appearing in search result ,i think this would be great for privacy
It would be good if material files support this
Below screenshot is for suggestion explaining its use
![Screenshot_20210215-191539](https://user-images.githubusercontent.com/75420364/107954689-99383880-6fbe-11eb-8672-9156fbf80fa6.png)
Look when creating .nopreview stops thumbnails in mixplorer  
Currently Material files doesnt respect .nopreview and .nosearch file
![Screenshot_20210215-191551](https://user-images.githubusercontent.com/75420364/107954775-b836ca80-6fbe-11eb-9a45-f2a201bfc140.png)

I hope that this feature request  is clear for you 
Thank you very much
